docs/dsp-tools-xmlupload.md

```diff
@@ -644,7 +644,7 @@ Example:
 
 ```xml
 <time-prop name=":hasTime">
-  <time>2019-10-23T13.45:12Z</time>
+  <time>2019-10-23T13:45:12Z</time>
```

Comment:
I never tested this, did you verify that it works like this?

Reply:
I have not tested it. I only saw the required format written at line 609: `The <time> element represents an exact datetime value in the form of yyyy-mm-ddThh:mm:ss.sssssssssssszzzzzz.`

Reply:
OK, I would recommend to test it to be sure that it works like stated (you never know with dsp-tools ;). Do you have the capacity to do that? Otherwise I can do it.

Reply:
Was tested and it worked -> https://github.com/dasch-swiss/082E-rosetta-scripts/pull/19